### ISSUE TITLE
Fix a11y violations reported by axe

### ISSUE THREAD

**Issue by wkoot** [ICTU/quality-time#6354]
See the `application_tests` artifact `build/a11y_violations.txt`. Found 6 accessibility violations:
```
Rule Violated:
aria-allowed-role - Ensures role attribute has an appropriate value for the element
	URL: https://dequeuniversity.com/rules/axe/3.1/aria-allowed-role?application=axeAPI
	Impact Level: minor
	Tags: cat.aria best-practice
	Elements Affected:
	1) Target: a[href$="about-us"]
		role listitem  is not allowed for given element
	2) Target: .three.left.wide:nth-child(2) > .link.list[role="list"] > a:nth-child(2)
		role listitem  is not allowed for given element
	3) Target: .three.left.wide:nth-child(2) > .link.list[role="list"] > a:nth-child(3)
		role listitem  is not allowed for given element
	4) Target: .three.left.wide:nth-child(2) > .link.list[role="list"] > a:nth-child(4)
		role listitem  is not allowed for given element
	5) Target: .three.left.wide:nth-child(5) > .link.list[role="list"] > a:nth-child(1)
		role listitem  is not allowed for given element
	6) Target: .three.left.wide:nth-child(5) > .link.list[role="list"] > a:nth-child(2)
		role listitem  is not allowed for given element
	7) Target: .three.left.wide:nth-child(5) > .link.list[role="list"] > a:nth-child(3)
		role listitem  is not allowed for given element
	8) Target: .three.left.wide:nth-child(5) > .link.list[role="list"] > a:nth-child(4)
		role listitem  is not allowed for given element


Rule Violated:
heading-order - Ensures the order of headings is semantically correct
	URL: https://dequeuniversity.com/rules/axe/3.1/heading-order?application=axeAPI
	Impact Level: moderate
	Tags: cat.semantics best-practice
	Elements Affected:
	1) Target: .three.left.wide:nth-child(2) > h4
		Heading order invalid


Rule Violated:
image-alt - Ensures <img> elements have alternate text or a role of none or presentation
	URL: https://dequeuniversity.com/rules/axe/3.1/image-alt?application=axeAPI
	Impact Level: critical
	Tags: cat.text-alternatives wcag2a wcag111 section508 section508.22.a
	Elements Affected:
	1) Target: .centered
		Element does not have an alt attribute
		aria-label attribute does not exist or is empty
		aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
		Element has no title attribute or the title attribute is empty
		Element's default semantics were not overridden with role="presentation"
		Element's default semantics were not overridden with role="none"


Rule Violated:
label - Ensures every form element has a label
	URL: https://dequeuniversity.com/rules/axe/3.1/label?application=axeAPI
	Impact Level: critical
	Tags: cat.forms wcag2a wcag332 wcag131 section508 section508.22.n
	Elements Affected:
	1) Target: input
		aria-label attribute does not exist or is empty
		aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
		Form element does not have an implicit (wrapped) <label>
		Form element does not have an explicit <label>
		Element has no title attribute or the title attribute is empty


Rule Violated:
landmark-one-main - Ensures the page has only one main landmark and each iframe in the page has at most one main landmark
	URL: https://dequeuniversity.com/rules/axe/3.1/landmark-one-main?application=axeAPI
	Impact Level: moderate
	Tags: cat.semantics best-practice
	Elements Affected:
	1) Target: html
		Page does not have a main landmark


Rule Violated:
region - Ensures all page content is contained by landmarks
	URL: https://dequeuniversity.com/rules/axe/3.1/region?application=axeAPI
	Impact Level: moderate
	Tags: cat.keyboard best-practice
	Elements Affected:
	1) Target: html
		Some page content is not contained by landmarks
```
